hack/build-supernode-image.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+curDir=$(cd "$(dirname "$0")" && pwd)
+SUPERNODE_SOURCE_HOME="${curDir}/../src/supernode"
+
+buildDockerImage() {
+    cd "${SUPERNODE_SOURCE_HOME}" || return
+    mvn clean package -DskipTests docker:build
+}
+
+buildDockerImage "$@"
```

Comment:
Try to click the key `Enter`, and you might not get this weird symbol in the end. So do with others.